VTEX - Logistics API.json

```diff
@@ -8160,17 +8211,88 @@
+                        "description": "OK",
+                        "content": {
+                            "application/json": {
+                                "schema": {
+                                    "type": "object",
+                                    "properties": {
+                                        "isActive": {
+                                            "type": "boolean",
+                                            "description": "Defines if the polygon is active (`true`) or inactive (`false`)."
+                                        },
+                                        "name": {
+                                            "type": "string",
+                                            "description": "[Polygon](https://help.vtex.com/en/tutorial/registering-geolocation--tutorials_138) ID, which is the same as the polygon name."
+                                        },
+                                        "geoShape": {
+                                            "type": "object",
+                                            "description": "Polygon geolocation coordinates.",
+                                            "properties": {
+                                                "coordinates": {
+                                                    "type": "array",
+                                                    "description": "List of geolocation coordinates that make part of the polygon.",
+                                                    "items": {
+                                                        "type": "array",
+                                                        "description": "Polygon geolocation coordinates, first longitude then latitude.",
+                                                        "items": {
+                                                            "type": "number",
+                                                            "description": "Longitude or latitude."
+                                                        }
+                                                    }
+                                                },
+                                                "type": {
+                                                    "type": "string",
+                                                    "description": "Polygon identification."
+                                                }
+                                            }
+                                        }
+                                    }
+                                },
+                                "example": {
+                                    "isActive": true,
+                                    "name": "ipanema_leblon",
+                                    "geoShape": {
+                                        "coordinates": [
+                                            [
+                                                [
```

Comment:
There is an extra `[]` in this sequence. Hence the spectral alert.

It should be 
"coordinates": [[-43.231716156005859, -22.990713500405295],[...],[...]]

instead of
"coordinates": [[[-43.231716156005859, -22.990713500405295],[...],[...]]]

Reply:
Nevertheless, the code has 3 levels of array.

Reply:
Ok, but then it means the schema is wrong. There should be one extra `items` layer.